### PR TITLE
fix(notifier): disables renew button upon renewal

### PR DESCRIPTION
**Pull request by vjgene**
resolves https://rsklabs.atlassian.net/jira/software/projects/RMKT/boards/60?selectedIssue=RMKT-829

https://user-images.githubusercontent.com/11134695/126683431-4a6851ae-30e0-4b58-b930-2cd2bbad05b1.mp4

### CODE REVIEW

File: src/components/pages/notifier/mypurchase/mapMyPurchases.tsx [hunk 1]
```diff
@@ -52,10 +52,9 @@ const mapMyPurchases = <V extends Function, R extends Function>(
 
     const expType = getExpirationType({ status, expirationDate }, plan)
     const { provider: providerAddress } = provider
-    const canRenew = (
-      status === SUBSCRIPTION_STATUSES.COMPLETED
-      || status === SUBSCRIPTION_STATUSES.EXPIRED
-    )
+    const canRenew = (status === SUBSCRIPTION_STATUSES.COMPLETED
+      || status === SUBSCRIPTION_STATUSES.EXPIRED)
```
Comment (jurajpiar) [inline src/components/pages/notifier/mypurchase/mapMyPurchases.tsx]:
```suggestion
    const canRenew = [
      SUBSCRIPTION_STATUSES.COMPLETED,
      SUBSCRIPTION_STATUSES.EXPIRED
    ].includes(status)
```

Comment (jurajpiar) [reply src/components/pages/notifier/mypurchase/mapMyPurchases.tsx]: or better still, use `const RENEWABLE_STATUS = [
      SUBSCRIPTION_STATUSES.COMPLETED,
      SUBSCRIPTION_STATUSES.EXPIRED
    ]` and `... RENEWABLE_STATUS.includes(status) ...`

File: src/components/pages/notifier/mypurchase/mapMyPurchases.tsx [hunk 2]
```diff
@@ -52,10 +52,9 @@ const mapMyPurchases = <V extends Function, R extends Function>(
...
+      && sessionStorage.getItem(id) === null
```
Comment (jurajpiar) [inline src/components/pages/notifier/mypurchase/mapMyPurchases.tsx]:
```suggestion
      && !sessionStorage.getItem(id)
```

**Review by itofarina — APPROVED** [review]
LGTM

**Pull request merged**